core-bundle/src/DependencyInjection/Compiler/RegisterFragmentsPass.php

```diff
@@ -28,24 +26,33 @@
 /**
  * Registers Contao fragments in the registry.
  *
- * For custom fragment tags, create your own compiler pass by extending this
- * class and replacing the process() method.
+ * For custom fragment tags, register your own compiler pass instance in your bundle.
  */
 class RegisterFragmentsPass implements CompilerPassInterface
 {
     use PriorityTaggedServiceTrait;
 
+    /**
+     * @var string
+     */
+    private $tag;
+
+    public function __construct(string $tag = null)
```

Comment:
I think I would prefer `array $tags = []` here. WDYT?

Reply:
That was my first approach. But there is really no reason for one instance to handle multiple tags, and extension devs would require to enter an array with most likely just one value …

Reply:
Agreed.